config/cli-config.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
this file is needed to tell to doctrine/migrations which entity maanger to use. Its name is hard coded in migrations